scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
OK, this needs to be reworked then; the release script assumes it's building the binaries